### DISCUSSION TITLE
LPD and Octave Shifts

### DISCUSSION THREAD

**Discussion by denisgottardello** [lenmus/lomse#397] [Q&A]
Hi, which is the way to increase or decrease the scale as in the picture?
![Screenshot_20240625_203726](https://github.com/lenmus/lomse/assets/43038562/a07dd45e-6701-4b19-a9b1-52c045e9d016)

**Comment by cecilios** [comment]
It is not possible in LDP to define octave shifts. Adding this feature
sholud not be complex as octave shifts are supported. But I do not have
plans to add this to LDP.  So, for now, octave shifts are only supported
for sources in MusicXML.

Adding this to LDP would probably follow the same approach as slurs but, as
said, I do not have plans in short term to add this to LDP

**Comment by cecilios** [comment]
If you really need this I could try to add this feature along the summer

**Comment by denisgottardello** [comment]
I don't know how to say thanks. Sure I'll use this feature to improve my music reading. I'm using lomse library in a free software for musicians. Up to now the software has downloaden rarely but in the future it could help many peolpe.

**Comment by cecilios** [comment]
I will add octave shifts to LDP. If in future exercises you will use more
music notation than the small set supported in LDP please consider moving
to MusicXML. I have no plans for adding more features to LDP as it will be
replicating other efforts such as MusicXML or it possible succersor MNX

**Comment by cecilios** [comment]
I have open issue #398 for this.

I would appreciate it if you could mark discussions you start as 'answered' when appropriate, to avoid them being left open. Thank you.

**Comment by cecilios** [comment]
Support for octave-shift lines in LDP added in PR#399
